frontend/packages/knative-plugin/src/utils/create-knative-utils.ts

```diff
@@ -103,6 +108,8 @@ export const getKnativeServiceDepResource = (
                   },
                 ],
               }),
+              imagePullPolicy: imgPullPolicy,
+              env,
```

Comment:
@invincibleJai since `Deployment` is not created in case of `KnativeService`, for pre-populating the deployment section trigger and env variables in edit forms you will need to make changes to `getDeploymentData` of `edit-application-utils` to get this data from `KnativeService`. I am pointing this out since deployment section will be visible in edit forms also but will not be pre-populated if trigger and env values exist. WDYT?

Reply:
Yeah thanks @divyanshiGupta , have updated here aef5f77f3163fcc5ebde541ddbbcfd7d05d5a1d0